JiraPS/Public/New-JiraIssue.ps1

```diff
@@ -119,7 +119,27 @@ function New-JiraIssue {
             $name = $_key
             $value = $Fields.$_key
 
-            if ($field = Get-JiraField -Field $name -Credential $Credential -Debug:$false) {
+            $fieldsMatchingName = Get-JiraField -Field $name -Credential $Credential -Debug:$false
+            if ($fieldsMatchingName.Count -gt 0) {
+                $field = $fieldsMatchingName[0]
+                if ($fieldsMatchingName.Count -gt 1) {
+                    $fieldsMatchingId = @($fieldsMatchingName | Where-Object {
+                        $each = $_
+                        $each.ID -eq $name
+                    })
+                    if ($fieldsMatchingId.Count -eq 1) {
```

Comment:
Unit test shows that this is not compatible with Windows Powershell (v5).
My best guess (without investigating) is that the `Where-Object` does not return an array and `.Count` does not exist.